test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -2928,6 +2946,9 @@ class test_ucp_sockaddr_protocols_diff_net_devices :
             value = "all";
         }
 
+        UCS_TEST_MESSAGE << peer_name << " using NET_DEVICES=" << value
+                         << (only_net_device ? std::string(" (") +
```

Comment:
1. do we really need std::string here? to_ip_str already returns string
2. use ( ) around + operation